.github/workflows/cleanup-envs.yaml

```diff
@@ -22,6 +22,10 @@ on:
         description: Whether to include environments that are not found (got deleted)
         type: boolean
         default: false
+      env-label:
+        description: The label prefix for automatic GitHub Actions
+        type: string
+        default: ${{ inputs.env-label }}
```

Comment:
shouldn't we keep the default to "auto-gha"? what does this auto referencing default do?